pages/Overview/RecentMatchTable.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <UiDefaultContainer class="mb-6 p-6">
+    <h2 class="text-xl font-bold text-slate-900 dark:text-white mb-4">{{ $t('recen') }}</h2>
+    <table class="w-full">
+      <TableHeader />
+      <tbody>
+        <tr v-for="(auction, index) in listingStore.recentMatches" :key="index"
+          class="border-b border-slate-200 dark:border-slate-700 hover:bg-slate-50 dark:hover:bg-slate-700/50">
+          <td class="px-4 py-3 text-sm text-slate-900 dark:text-white">
+            <div class="flex items-center space-x-3">
+              <img :src="auction.image ?? ''" alt="Auction thumbnail" class="w-10 h-10 rounded-md object-cover" />
+              <div>
+                <p class="font-medium">{{ auction.title }}</p>
+                <p class="text-xs text-slate-500 dark:text-slate-400">{{ auction.marketplace }}</p>
+              </div>
+            </div>
+          </td>
+          <td class="px-4 py-3 text-sm text-slate-900 dark:text-white">{{ filters.getSimplifiedFilters[auction.filter ??
+            0] }}
+          </td>
+          <td class="px-4 py-3 text-sm font-medium text-slate-900 dark:text-white">{{ auction.price }}</td>
+          <td class="px-4 py-3 text-sm text-slate-500 dark:text-slate-400">{{ timeAgo(auction.matchedAt) }}</td>
+          <td class="px-4 py-3 text-sm">
+            <div class="flex items-center space-x-2 ml-2.5">
+              <a :href="auction.url ?? ''" target="_blank"
+                class="p-1 rounded-md hover:bg-slate-100 dark:hover:bg-slate-700" title="View auction">
+                <Icon name="tabler:external-link" class="size-5 text-slate-400" />
+              </a>
+
+            </div>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </UiDefaultContainer>
+</template>
+
+<script setup lang="ts">
+import TableHeader from './TableHeader.vue';
+
+const listingStore = useListingStore()
+const filters = useFilterStore()
+
+function timeAgo(timestamp: string): string {
+  const now = new Date();
+  const past = new Date(timestamp);
+  const diff = (now.getTime() - past.getTime()) / 1000;
+  if (diff < 900) {
```

Comment:
if you only do  a return inside a if, you dont need a else if next. Do it like this:

if(a){
return 1
}

if(b){
return 2
}
...